homeassistant/components/opsgenie/notify.py

```diff
@@ -0,0 +1,82 @@
+"""Opsgenie platform for notify component."""
+import logging
+
+import opsgenie_sdk
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    ATTR_DATA,
+    ATTR_TARGET,
+    ATTR_TITLE,
+    ATTR_TITLE_DEFAULT,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+from homeassistant.const import CONF_API_KEY, CONF_URL
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ALIAS = "alias"
+ATTR_ACTIONS = "actions"
+ATTR_TAGS = "tags"
+ATTR_PRIORITY = "priority"
+ATTR_ENTITY = "entity"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Required(CONF_API_KEY): cv.string, vol.Required(CONF_URL): cv.string}
+)
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

This integration needs to be refactored first to use a configuration flow and config entries first.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

See our developer documentation on how to get started creating a configuration flow for this integration:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

This integration does bring in a notification platform, which needs to use `discovery.async_load_platform` to set up the notify platform when using the configuration flow. As a reference implementation, you could look at the Tibber integration.

As these changes often involve a bit of work and some significant shift in the current code, we will close this PR for now.

We (and our community!) would really appreciate it if you could start on the refactoring of this integration in a new PR.

Thanks already! :+1: